extension-manifest-v3/src/utils/opera.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+import { setCookie } from '/utils/api.js';
+
+const TEST_COOKIE_NAME = `ghostery:opera:cookie:test:${Date.now()}`;
+
+let isSupported = undefined;
```

Comment:
After enabling SERP pages, the browser reloads the extension. We can use that fact to make a check only once.